staging/src/k8s.io/apiserver/plugin/pkg/authenticator/token/webhook/webhook.go

```diff
@@ -109,7 +121,22 @@ func (w *WebhookTokenAuthenticator) AuthenticateToken(ctx context.Context, token
 	// WithExponentialBackoff will return tokenreview create error (tokenReviewErr) if any.
 	if err := webhook.WithExponentialBackoff(ctx, w.retryBackoff, func() error {
 		var tokenReviewErr error
-		result, tokenReviewErr = w.tokenReview.Create(ctx, r, metav1.CreateOptions{})
+		var statusCode int
+
+		start := time.Now()
+		result, statusCode, tokenReviewErr = w.tokenReview.Create(ctx, r, metav1.CreateOptions{})
+		latency := time.Now().Sub(start)
+
+		if statusCode != 0 {
+			w.metrics.RequestTotal(ctx, strconv.Itoa(statusCode), "POST")
+			w.metrics.RequestLatency(ctx, latency.Seconds(), strconv.Itoa(statusCode), "POST")
+			return tokenReviewErr
+		}
+
+		if tokenReviewErr != nil {
+			w.metrics.RequestTotal(ctx, "<error>", "POST")
```

Comment:
"network-error"? Do we have similar cases elsewhere for such a string?

Reply:
we already use it in `client-go` - https://github.com/kubernetes/kubernetes/blob/c039b02fa7281fc061455e23b6530ed8b4d19645/staging/src/k8s.io/client-go/rest/request.go#L739